### PR TITLE
change to colorScheme = NULL fixing #33

### PR DESCRIPTION
**Pull request by BradyAJohnston**
Fixes error with colorScheme = "default" that overrides user-specified colours.

Fixes bug raised in #33.

### CODE REVIEW

File: R/style.R [hunk 1]
```diff
@@ -243,7 +243,7 @@ m_style_label <- function(
 #'   m_zoom_to()
 #' @export
 m_style_line <- function(
-                         colorScheme = "default",
+                         colorScheme = NULL,
```
Comment (swsoyee) [inline R/style.R:246]: How about also fix the documentations about `colorScheme`?

```
#' @param colorScheme Specify scheme to color the atoms by. Default is
#' "default". Other choies are "Carbon", ssPyMOL", "ssJmol", "Jmol",
#' "default", "amino",
#' "shapely", "nucleic", "chain", "chainHetatm", "prop".
```

- Maybe there is a typo in `choies`?
- A quotation is missing before `ssPyMOL`

**Review by swsoyee — CHANGES_REQUESTED** [review]
It would be better to add the fix to NEWS.md
https://github.com/swsoyee/r3dmol/blob/master/NEWS.md

I will release the v0.1.3 to CRAN if there are not other fixes recently.

**Review by swsoyee — APPROVED** [review]
Since the PR is LGTM, but I am very sorry for taking the liberty of fixing some typos in the documentation. I'll be able to merge it when the check passes. Thanks a lot for fixing this bug!

**Comment by BradyAJohnston** [comment]
Thanks for fixing the minor stuff - it seems that I was having trouble with generating the documentation etc for some reason. Thanks for the fix and thanks for the merge!

**Pull request merged**